kernel/kernel-impl/src/main/java/org/sakaiproject/tasks/impl/TaskServiceImpl.java

```diff
@@ -187,8 +219,29 @@ public void completeUserTaskByReference(String reference, List<String> userIds)
 
     @Transactional
     public void removeTask(Task task) {
-
```

Comment:
Please leave these blank lines in. I put them in all my code as I think it makes methods easier to read, less cramped.

Reply:
I personally hate blank lines under curly braces. I think this don't affect readability at all. I've never read java code guidelines that recommend this...

Reply:
How can you hate a blank line? Please *don't* go and remove them from the rest of the class though.

Reply:
Intellij IDEA has the option, so blank lines are a thing,  not just my weirdness.

https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000363879-Blank-line-after-function-class-definition